README.md

```diff
@@ -0,0 +1,65 @@
+# Local development
+## Setup 
+1. [Install Docker](https://docs.docker.com/install/) 
+1. [Install Docker Compose](https://docs.docker.com/compose/install/)
+1. [Install 10updocker](https://10up.github.io/wp-local-docker-docs/installation/)
+1. Linux Only: Run the command below and restart
+    ```bash
+    sudo usermod -a -G docker $USER
+    ```
+1. Configure 10updocker
+    ```bash
+    10updocker configure
+    ```
+    When asked `What directory would you like WP Local Docker to create environments within? ` choose the directory above the ctoec repository. 
+1. Create new env
```

Comment:
Docker must be running before this step.

Use `ctoec.test` as the hostname, and accept all the defaults for other configuration options.